packages/components/src/Tokens/sell/TokensTabSell.tsx

```diff
@@ -0,0 +1,59 @@
+import * as React from "react";
+import { TokensTabSellActive } from "./TokensTabSellActive";
+import { TokensTabSellComplete } from "./TokensTabSellComplete";
+import { TokensTabSellUnlock } from "./TokensTabSellUnlock";
+import { ComingSoon } from "../TokensStyledComponents";
+import { FeatureFlag } from "../../features/FeatureFlag";
+
+export interface TokensTabSellProps {
+  network: string;
+}
+
+export interface TokensTabSellStates {
+  isSellComplete: boolean;
+}
+
+export class TokensTabSell extends React.Component<TokensTabSellProps, TokensTabSellStates> {
+  public constructor(props: TokensTabSellProps) {
+    super(props);
+    this.state = {
+      isSellComplete: false,
+    };
+  }
+
+  public render(): JSX.Element | null {
+    const { isSellComplete } = this.state;
+    // TODO(sarah): check token controller.
+    const isTokenUnlocked = true;
```

Comment:
this should check the whitelist there's a helper in dapp helpers which should probably be moved to utils `getUnlockedWhitelist(userAccount)`